packages/components-css/card/index.scss

```diff
@@ -0,0 +1,10 @@
+.frameless-card {
+  border-radius: var(--frameless-border-radius-sm);
```

Comment:
ik verwacht hier een **component** token, dus bijvoorbeeld
`var(--frameless-card-border-radius)` die kan in de JSON dan bijvoorbeeld verwijzen naar de **brand** token:

```json
{
  frameless: {
    card: {
      border-radius: {
        value: "{frameless.border-radius.sm}"
      }
    }
  }
}
```